src/handleAction.js

```diff
@@ -7,19 +7,14 @@ export default function handleAction(type, reducers, defaultState) {
     ? type.toString()
     : type;
 
+  const nextReducer = isFunction(reducers) ? reducers : reducers.next;
+  const throwReducer = isFunction(reducers) ? reducers : reducers.throw;
```

Comment:
Oh that's perfect. I was actually wishing I could do that but I couldn't work out a clean way to do it... I didn't know you could do `[a, b] = ...` in addition to `{a, b} = ...` Will update, probably with:

```
const [nextReducer, throwReducer] = isFunction(reducers) ?
    [reducers, reducers] :
    [reducers.next, reducers.throw]
```